gnuradio-runtime/include/pmt/pmt.h

```diff
@@ -642,9 +642,13 @@ c64vector_writable_elements(pmt_t v, size_t& len); //< len is in elements
  * ------------------------------------------------------------------------
  */
 
-//! Return true if \p obj is a dictionary (warning: also returns true for a pair)
+//! Return true if \p obj is a dictionary
 PMT_API bool is_dict(const pmt_t& obj);
 
+//! Return a newly allocated dict whose car is a key-value pair \p x and whose cdr is a
+//! dict \p y.
```

Comment:
This should fail when `y` is not a dict, and `x` is not a key/value pair. Please document the exception or error case if they're not.

Reply:
good call and this has been fixed